VSharp.Explorer/AISearcher.fs

```diff
@@ -361,8 +361,12 @@ type internal AISearcher(oracle: Oracle, aiAgentTrainingMode: Option<AIAgentTrai
                         let parentOf = Array.zeroCreate (2 * numOfParentOfEdges)
                         let shapeOfHistory = [| 2L; numOfHistoryEdges |]
                         let historyIndex_vertexToState = Array.zeroCreate (2 * numOfHistoryEdges)
-                        let shapeOfPcToState = [| 2L; gameState.States.Length |]
-                        let index_pcToState = Array.zeroCreate (2 * gameState.States.Length)
+
+                        let pathConditionNum = 
```

Comment:
It would be better to move ```pathConditionNum``` calculation closer to [this line](https://github.com/PySymGym/VSharp/pull/98/files#diff-357de517c457679b5d3b871557406c83d08447364fbe33e398226dcc1cfa6617R317) to prevent redundant iterations over states. Look at the ```numOfParentOfEdges``` calculation.